modules/timer/antara/gaming/timer/fps.hpp

```diff
@@ -16,13 +16,14 @@
 
 #pragma once
 
```

Comment:
May be we can rename the file ?

Reply:
fps is fine imo, easier to find and understand than tps. file also has fps stuff in it